apps/emqx/src/emqx_modules.erl

```diff
@@ -82,10 +82,11 @@ reload(emqx_mod_acl_internal) ->
     Env = proplists:get_value(emqx_mod_acl_internal, Modules, undefined),
     case emqx_mod_acl_internal:reload(Env) of
         ok ->
-            ?LOG(info, "Reload ~s module successfully.", [emqx_mod_acl_internal]);
+            ?LOG(info, "Reload ~s module successfully.", [emqx_mod_acl_internal]), ok;
```

Comment:
doesn't `?LOG(...)`  return `ok`.
either case, it's better to return `ok` outside of the case clauses.
i.e. add `ok;` to line 89.